### ISSUE TITLE
Adding tags for SEO

### ISSUE THREAD

**Issue by dennyabrain** [tattle-made/website#108]
For every page or post that is published on our site, we must add relevant tags in the metadata for that post. We'll need to write some code so that gatsby puts those tags into the html head element so that search engine crawlers can index it.

The scope of this issue includes : 

- [ ] Build the feature for taking tags from the markdown frontmatter and putting it in the html head element
- [ ] Add tags to all the existing content and verify if they are appearing in the published pages
- [ ] Documenting the process in the README for future contributors

**Comment by dennyabrain** [comment]
The scope of this issue includes : 

- [ ] Build the feature for taking tags from the markdown frontmatter and putting it in the html head element
- [ ] Add tags to all the existing content and verify if they are appearing in the published pages
- [ ] Documenting the process in the README for future contributors

**Comment by aatmanvaidya** [comment]
gatsby version 4.19.0 or higher is required to add SEO, else we will have to use plugins. - https://www.gatsbyjs.com/docs/how-to/adding-common-features/adding-seo-component/
https://www.digitalocean.com/community/tutorials/how-to-boost-seo-using-gatsby-s-seo-component-and-gatsby-react-helmet

**Comment by maanasb01** [comment]
Hello, are the tags to be included in the `<meta name="keywords">` tag? I tried this issue and was able to include the tags for each blog post page in the `<meta name="keywords">` tag. I created a Map for each slug and its tags and then added the keywords in the `<meta name="keywords">` tag based on the page's slug. For the current page's slug, I am extracting it from the path coming from the `useLocation` hook (which was already being used in Seo.js ). I have done this for blog post pages only (as the path check was already being done for "/blog/", so I just extracted the slug from there for all blog posts). What other pages would require tags?
